docs/Guide.md

```diff
@@ -57,7 +57,7 @@ Ignite comes with some prebuilt, flexible, and customizable components. Unlike m
 
 Ignite works fine with other component libraries, but the built-in component system works the best for custom-designed apps.
 
-Check out the [Components](./components.md) documentation.
+Check out the [Components](./Components.md) documentation.
```

Comment:
looked at all other references to `components.md` across codebase and they are all pointing at `Components.md`, just this one was not correct.